<launch>
    <node pkg="ros_interface" type="robot_server.py" name="robot_server" output="screen"/>
    <node pkg="ros_interface" type="jaco.py" name="jaco_interface" output="screen"/>
    <arg name="robotType" value="j2n7s300"/> 
    <arg name="feedback_publish_rate" value="0.01"/>
    <include file="$(find kinova_bringup)/launch/kinova_robot.launch"/>
</launch>
